cw_core/lib/transaction_direction.dart

```diff
@@ -20,4 +20,14 @@ TransactionDirection parseTransactionDirectionFromNumber(String raw) {
     default:
       throw Exception('Unexpected token: raw for TransactionDirection parseTransactionDirectionFromNumber');
   }
-}
\ No newline at end of file
+}
+int parseTransactionDirectionFromType(TransactionDirection raw) {
+  switch (raw) {
+    case TransactionDirection.incoming:
+      return 0;
+    case TransactionDirection.outgoing:
+      return 1;
+    default:
+      throw Exception('Unexpected token: raw for TransactionDirection parseTransactionDirectionFromType');
+  }
+}
```

Comment:
this can be accessed from the `index` attribute of the enum instead of defining a separate function, plus there won't be a case where it throws an exception as it will always have a value